databricks/koalas/series.py

```diff
@@ -348,6 +348,25 @@ def index(self):
             raise KeyError('Currently supported only when the Column has a single index.')
         return self._kdf.index
 
+    @property
+    def is_unique(self):
+        """
+        Return boolean if values in the object are unique
+
+        .. note:: this is an expensive operation that has to count all values in the Series.
+
+        Returns
+        -------
+        is_unique : boolean
+
+        >>> ks.Series([1, None, None]).is_unique
+        False
+        >>> ks.Series([1, 2, 3, None]).is_unique
+        True
+        """
+        sdf = self._kdf._sdf.select(self._scol)
```

Comment:
the way i'd do this is to do a single pass, e.g.

select(count(col), countDistinct(col))

and get the two values back and compare them at once.

Reply:
I actually tried one pass first. The problem is countDistinct ignore null and `expr` with COUNT(DISTINCT *) ignore null by design as well.

Reply:
It seems like we are fine?

```
>>> ks.DataFrame([1, 2, 3, 4, None, None]).to_spark().selectExpr("count(distinct `0`)").show()
+-----------------+
|count(DISTINCT 0)|
+-----------------+
|                4|
+-----------------+
```

This is the same as pandas:
```
>>> pd.Series([1, 2, 3, 4, None, None]).nunique()
4
```

Reply:
Yes, I will stick to the original one.

Reply:
Actually, Pandas seems using:

```
>>> pd.Series([1, 2, 3, 4, None, None]).nunique(dropna=False)
5
```